packages/base/src/commands.ts

```diff
@@ -878,6 +878,56 @@ export function addCommands(
     icon: targetWithCenterIcon
   });
 
+  commands.addCommand(CommandIDs.newDrawVectorLayer, {
+    label: trans.__('Create New Draw Vector Layer'),
+    isToggled: () => {
+      const selectedLayer = getSingleSelectedLayer(tracker);
+      if (!selectedLayer) {
+        return false;
+      }
+
+      const canDrawVectorLayer = ['VectorLayer'].includes(selectedLayer.type);
```

Comment:
For the first version, let's allow only editing vector layers with a `data` property (embedded data in the jgis file).

Saving edits of files backed by external url or local path requires more discussion (for those files, we could save the diff, and allow exporting the result to a new file on the server).

```suggestion
      const canDrawVectorLayer = selectedSource.type === 'GeoJSONSource' && selectedSource.data;
```